src/webextension/manage/components/homepage.js

```diff
@@ -3,49 +3,32 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import { withLocalization } from "fluent-react";
+import { Localized } from "fluent-react";
 import PropTypes from "prop-types";
 import React from "react";
 
 import styles from "./homepage.css";
-import lockie from "./lockie.txt";
-
-function printableLength(s) {
-  return s.replace(/(\u2068|\u2069)/g, "").length;
-}
-
-function speechBubble(text) {
-  const lines = text.split("\n");
-  const innerWidth = Math.max(...lines.map(printableLength), 1);
-
-  const result = [
-    " " + "_".repeat(innerWidth + 4),
-    "/" + " ".repeat(innerWidth + 4) + "\\",
-    ...lines.map((i) => {
-      return "|  " + i + " ".repeat(innerWidth - printableLength(i)) + "  |";
-    }),
-    "\\__   " + "_".repeat(innerWidth - 1) + "/",
-    "   \\ /",
-    "    v",
-  ];
-  return result.join("\n");
-}
 
 function Homepage({count, getString}) {
+  const imgSrc = browser.extension.getURL("/images/lockie_v2.svg");
+
   let key;
   if (count === 0) {
-    key = "homepage-no-passwords";
-  } else if (count < 10) {
-    key = "homepage-under-10-passwords";
-  } else if (count < 50) {
-    key = "homepage-under-50-passwords";
+    key = "homepage-no-passwords-title";
   } else {
-    key = "homepage-over-50-passwords";
+    key = "homepage-has-passwords-title";
   }
 
   return (
-    <pre className={styles.homepage}>{
-      speechBubble(getString(key, {count})) + "\n" + lockie
-    }</pre>
+    <article className={styles.homepage}>
+      <img src={imgSrc}/>
```

Comment:
nit: Do we want to add an `alt=""` tag here, or are we doing an a11y pass later?

Reply:
I'm not sure what we'd say for this image specifically. We kinda intentionally don't want to "spoil" the visual joke that we're using a Loch-ness (lock) monster. 😬

Reply:
Let's do as @pdehaan suggests. `alt=""` (literally using the empty string, not filling it in like `alt="The Lockie Empire will reign for 1,000 years"`) is the recommended way of indicating that an image is purely-decoratinve.

Reply:
>See: https://www.w3.org/WAI/tutorials/images/decorative/

Ah! Sorry I misunderstood. TIL...

![giphy](https://user-images.githubusercontent.com/49511/32581615-dc228cec-c4a7-11e7-99d5-1f67030c5fbb.gif)